ts/components/NavBarLabel.tsx

```diff
@@ -61,6 +61,7 @@ const NavBarLabel: React.FunctionComponent<Props> = (props: Props) => {
   const locale: Locales = preferredLanguage.fold(I18n.locale, l => l);
   const label = getLabel(routeName, locale);
   const maybeOrder = fromNullable(routeOrder.get(routeName as Routes));
+  const isSelected = options.focused ? I18n.t("navigation.selected") : "";
```

Comment:
```suggestion
  const isSelected = options.focused ? `${I18n.t("navigation.selected")}, ` : "";
```
A really trivial little thing that has no effect on pronunciation, but just to keep a clean string when the item is not selected.
<img width="210" alt="Schermata 2021-07-12 alle 14 32 26" src="https://user-images.githubusercontent.com/26501317/125288039-fd68e980-e31d-11eb-8566-700afb6cc3d2.png">

Reply:
Fix PR has suggested ✅